### PR TITLE
Migrate to relay modern instead of Apollo client

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,21 +79,21 @@
   },
   "dependencies": {
     "@kiwicom/orbit-components": "^0.0.0-rc1",
-    "apollo-boost": "0.1.4",
     "axios": "^0.18.0",
-    "graphql": "0.13.2",
     "babel-runtime": "^6.26.0",
     "i18next": "10.5.0",
+    "isomorphic-fetch": "2.2.1",
```
Comment (git-toni) [inline package.json]: Wouldn't `axios` be enough?

**Comment by git-toni** [comment]
I think this PR should include the `get-graphql-schema` package.json script to retrieve the updated version of the schema from the graphql repo endpoint.

**Comment by jaroslav-kubicek** [comment]
I added `graphql-cli` instead for the same purpose, which should be better choice, cc @git-toni

File: package.json (hunk 1 above)
Comment (jaroslav-kubicek) [reply package.json]: `axios` is actually adding additional functionality which we don't need - transformers, interception or cancellation

**Comment by git-toni** [comment]
@jaroslav-kubicek  Yep, sorry, missed the `.graphqlconfig` settings. Much better like this I agree :)

File: package.json (hunk 1 above)
Comment (git-toni) [reply package.json]: Agree, then we should replace all `axios` uses with polyfilled `fetch`. I'll raise  an issue for that

**Pull request merged**